modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/accessor/index/ExperimentUserIndexAccessor.java

```diff
@@ -40,6 +41,10 @@
     @Query("select * from experiment_user_index where user_id = ? and app_name = ? and context = ?")
     Result<ExperimentUserByUserIdContextAppNameExperimentId> selectBy(String userId, String appName, String context);
```

Comment:
Need to keep this method as it is still being used in other places other than batch-assignment